configure.ac

```diff
@@ -16,10 +16,13 @@ AM_MAINTAINER_MODE
 
 # Checks for programs.
 AC_PROG_CC
+AC_PROG_CXX
```

Comment:
So this will now require a working C++ compiler even if angpow is not used? Can we wrap this only if angpow is downloaded?

Reply:
Hi Anze. I have just tried to make it optional, but then the autoreconf complains because of the CXX flags defined in Makefile.am for the check-cpp method. Any idea on how to go forward ?